private/buf/buffetch/internal/internal.go

```diff
@@ -704,18 +766,27 @@ type GetBucketOption func(*getBucketOptions)
 
 // WithGetBucketTerminateFileNames only applies if subdir is specified.
 //
-// This says that if given a subdir, ascend directories until you reach
+// The terminate files are organized as a slice of slices of file names.
+// Priority will be given to the first slice of terminate file names, which will be workspace
+// configuration files. The second layer of priority will be given to modules.
+//
+// This says that for a given subdir, ascend directories until you reach
 // a file one of these names, and if you do, the returned bucket will be
```

Comment:
nit: typo from the past
```suggestion
// a file with one of these names, and if you do, the returned bucket will be
```